src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -425,17 +428,25 @@ def _encode_prompt(
 
         return prompt_embeds
 
-    def run_safety_checker(self, image, device, dtype):
-        if self.safety_checker is not None:
-            safety_checker_input = self.feature_extractor(self.numpy_to_pil(image), return_tensors="pt").to(device)
+    def run_safety_checker(self, image, device, dtype, output_type="pil"):
+        if self.safety_checker is None or output_type == "latent":
+            has_nsfw_concept = False
```

Comment:
Previously, we would set the `has_nsfw_concept` to None, when we couldn't run the safety checker. False would only be returned if the safety_checker said definitively no. Did we decide to change that?

Reply:
yeah, I think I asked @patrickvonplaten that - we can double-check with him again I guess

Is there a use case `None` and `False` that would make a difference, in terms of how we use the pipeline? I understand they mean different things

Reply:
Happy to leave at `None` here as it might be better for backwards comp